docker-compose.sample.yml

```diff
@@ -86,6 +86,9 @@ services:
     volumes:
       - ${VOLUMES_ROOT_PATH}/mail:/var/mail
       - ${VOLUMES_ROOT_PATH}/traefik/acme:/etc/letsencrypt/acme
+      # Comment out the line below, when you want whitelist some IP Addresses or domains in Postfix (please check the documentation)
+      # - ${VOLUMES_ROOT_PATH}/<to_your_directory_where_the>/rbl_override:/etc/postfix/rbl_override
```

Comment:
May be we should prescribe what `to_your_directory_where_the` is. How about `${VOLUMES_ROOT_PATH}/postfix/rbl_override` ? It may be easier to support if people start with uniform default paths.

Reply:
sounds good, but the user needs to create manually the folder and the file.